src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,22 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
+        String login = "bob";
+        String password = "1234";
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login(login, password);
+            logger.info("User '{}' logged in successfully", login);
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Failed login attempt for user '{}': {}", login, e.getMessage(), e);
             return;
         }
         OrderService orderService = new OrderServiceImpl();
```

Comment:
Ensure that `OrderServiceImpl` correctly implements the `OrderService` interface and that it is properly configured to handle order operations.